perf-next/index.js

```diff
@@ -13,18 +13,18 @@ function intialize() {
 
   // [START perf_initialize_app]
   // TODO: Replace the following with your app's Firebase project configuration
-  // See: https://firebase.google.com/docs/web/setup#config-object
+  // See: https://firebase.google.com/docs/web/learn-more#config-object
   const firebaseConfig = {
     // ...
   };
 
   // Initialize Firebase
-  initializeApp(firebaseConfig);
+  const app = initializeApp(firebaseConfig);
   // [END perf_initialize_app]
 
   // [START perf_singleton]
   // Initialize Performance Monitoring and get a reference to the service
-  const perf = getPerformance();
+  const perf = getPerformance(app);
```

Comment:
This was an intentional change, but it's fine either way.

Reply:
@davideast said that he preferred being more explicit with the demo snippet to ensure folks know how it's actually working, but then folks can do whatever they want in their actual code.

Reply:
🤷 well a few months ago we removed this from all other snippets where possible:
https://github.com/firebase/snippets-web/pull/135